src/fidesops/service/connectors/sql_connector.py

```diff
@@ -340,8 +343,33 @@ def create_client(self) -> Engine:
         """Returns a SQLAlchemy Engine that can be used to interact with a MicrosoftSQLServer database"""
         config = MicrosoftSQLServerSchema(**self.configuration.secrets or {})
         uri = config.url or self.build_uri()
-        return create_engine(
-            uri,
-            hide_parameters=self.hide_parameters,
-            echo=not self.hide_parameters,
-        )
+        return create_engine(uri, hide_parameters=False, echo_pool="debug")
```

Comment:
We probably want to set `hide_parameters` and `echo` back to their original values here?

Reply:
good catch!